config/initializers/okcomputer.rb

```diff
@@ -1,58 +1,61 @@
-require "net/https"
-require "uri"
-
-class AvalonCheck < OkComputer::Check
-  def time_limit
-    5
-  end
+if defined? OkComputer
```

Comment:
@phuongdh I understood the other changes, but this one. I have 2 questions;
1. What does this do?
2. What happens if this fails? Does it fall back to `about-page` implementation?

Reply:
This is needed because we only use okcomputer in production, so it's not available in development & test and would break the app without this check.